### PR TITLE
Events 500

### PR DESCRIPTION
**Pull request by andrewpollack**

### CODE REVIEW

File: draft/2023-06-21-this-week-in-rust.md [hunk 1]
```diff
@@ -156,8 +142,12 @@ Rusty Events between 2023-06-21 - 2023-07-19 🦀
     * [**Research Computing With The Rust Programming Language - Tim McNamara**](https://www.eventbrite.com/e/research-computing-with-the-rust-programming-language-tickets-637648623197?aff=ebdssbdestsearch)
 * 2023-06-27 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/dallas-rust/)
     * [**Last Tuesday**](https://www.meetup.com/dallas-rust/events/qndgwsyfcjbkc/)
+* 2023-06-14 | Virtual (Cardiff, UK) | [Rust and C++ Cardiff](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/)
```
Comment (mariannegoldin) [inline draft/2023-06-21-this-week-in-rust.md:145]:
```suggestion
* 2023-06-28 | Virtual (Cardiff, UK) | [Rust and C++ Cardiff](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/)
```

File: draft/2023-06-21-this-week-in-rust.md [hunk 2]
```diff
@@ -168,26 +158,30 @@ Rusty Events between 2023-06-21 - 2023-07-19 🦀
...
-* 2023-06-20 | Montpellier, FR | [Montpellier Rust Meetup](https://www.meetup.com/montpellier-rust-meetup/)
-    * [**Meetup Rust Juin 2023**](https://www.meetup.com/montpellier-rust-meetup/events/294159602/)
 * 2023-06-22 | Vienna, AT | [Papers We Love Vienna](https://www.meetup.com/papers-we-love-vienna/)
     * [**June: Data and Ownership in Rust**](https://www.meetup.com/papers-we-love-vienna/events/293974147)
 * 2023-06-22 | Bern, CH | [Rust Bern](https://www.meetup.com/de-DE/rust-bern/)
     * [**Second Rust Bern Meetup**](https://www.meetup.com/de-DE/rust-bern/events/293619228/)
 * 2023-06-22 | Wrocław, PL | [Rust Wrocław](https://www.meetup.com/rust-wroclaw/)
     * [**Rust Meetup #34**](https://www.meetup.com/rust-wroclaw/events/294176450/)
+* 2023-06-27 | Bucharest, RO | [Rust Lang Bucharest Meetup](https://www.meetup.com/rust-lang-bucharest-meetup/)
+    * [**Rust Bucharest Meetup #2***](https://www.meetup.com/rust-lang-bucharest-meetup/events/294204963/)
```
Comment (mariannegoldin) [inline draft/2023-06-21-this-week-in-rust.md:184]:
```suggestion
    * [**Rust Bucharest Meetup #2**](https://www.meetup.com/rust-lang-bucharest-meetup/events/294204963/)
```

**Review by mariannegoldin — APPROVED** [review]
Looks great! Approved with 2 corrections.

**Comment by andrewpollack** [comment]
Awesome spots, thank you!

**Pull request merged**